adapters/dax/dax.go

```diff
@@ -4,94 +4,59 @@ import (
 	"encoding/json"
 	"encoding/xml"
 	"fmt"
-	"github.com/prebid/prebid-server/config"
-	"net/http"
-	"net/url"
-	"strconv"
-	"strings"
-	"time"
-
 	"github.com/mxmCherry/openrtb"
 	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/config"
 	"github.com/prebid/prebid-server/errortypes"
 	"github.com/prebid/prebid-server/openrtb_ext"
+	"net/http"
+	"strconv"
 )
 
 type DaxAdapter struct {
 	endpoint string
 }
 
-func (adapter *DaxAdapter) MakeRequests(request *openrtb.BidRequest, reqInfo *adapters.ExtraRequestInfo) ([]*adapters.RequestData, []error) {
-	numImps := len(request.Imp)
+func (adapter *DaxAdapter) MakeRequests(request *openrtb.BidRequest, requestInfo *adapters.ExtraRequestInfo) ([]*adapters.RequestData, []error) {
 
-	requests := []*adapters.RequestData{}
+	impressions := request.Imp
+	adapterRequests := make([]*adapters.RequestData, 0, len(impressions))
+	errs := make([]error, 0, len(impressions))
 
-	errors := []error{}
-
-	for i := 0; i < numImps; i++ {
-		imp := request.Imp[i]
+	if impressions == nil || len(impressions) == 0 {
```

Comment:
`impressions == nil` check can be removed, `len(impressions) == 0` is enough.

Reply:
good point, removed it

Reply:
Changes are pushed